src/python/grpcio/grpc/_channel.py

```diff
@@ -747,21 +850,21 @@ def __call__(  # pylint: disable=too-many-locals
         if serialized_request is None:
             state = _RPCState((), (), (), grpc.StatusCode.INTERNAL,
                               'Exception serializing request!')
-            raise _Rendezvous(state, None, None, deadline)
+            raise _RpcError(state)
 
         state = _RPCState(_UNARY_STREAM_INITIAL_DUE, None, None, None, None)
         call_credentials = None if credentials is None else credentials._credentials
         initial_metadata_flags = _InitialMetadataFlags().with_wait_for_ready(
             wait_for_ready)
         augmented_metadata = _compression.augment_metadata(
             metadata, compression)
-        operations_and_tags = ((
-            (cygrpc.SendInitialMetadataOperation(augmented_metadata,
-                                                 initial_metadata_flags),
-             cygrpc.SendMessageOperation(serialized_request, _EMPTY_FLAGS),
-             cygrpc.SendCloseFromClientOperation(_EMPTY_FLAGS),
-             cygrpc.ReceiveStatusOnClientOperation(_EMPTY_FLAGS)), None),) + (((
-                 cygrpc.ReceiveInitialMetadataOperation(_EMPTY_FLAGS),), None),)
+        operations_and_tags = (
+            (((cygrpc.SendInitialMetadataOperation(augmented_metadata,
+                                                   initial_metadata_flags),
+               cygrpc.SendMessageOperation(serialized_request, _EMPTY_FLAGS),
+               cygrpc.SendCloseFromClientOperation(_EMPTY_FLAGS)), None),) +
+            (((cygrpc.ReceiveStatusOnClientOperation(_EMPTY_FLAGS),), None),) +
+            (((cygrpc.ReceiveInitialMetadataOperation(_EMPTY_FLAGS),), None),))
```

Comment:
Last time I think I also commented on this. Personally, I think splitting `operations_and_tags` helps readability:
```Python
send_operations_and_tags = ...
receive_status_operations_and_tags = ...
receive_initial_metadata_operations_and_tags = ...
operations_and_tags = [
    send_operations_and_tags,
    receive_status_operations_and_tags,
    receive_initial_metadata_operations_and_tags,
]
```